yaqd_andor/atmcd_acquisition.py

```diff
@@ -0,0 +1,6 @@
+import numpy as np
+
+
+class Acquisition:
+    def __init__(self) -> None:
+        super().__init__()
```

Comment:
does this have a point?

Reply:
Probably not, it came with the SDK2 and I just left it in.

Reply:
I would remove it.  It is not used anywhere that I can see...